src/message-box.scss

```diff
@@ -100,7 +101,7 @@ $message-box-states: (
       }
 
       .#{$block}__title {
-        @include fd-icon-base () {
+        @include fd-icon-base ("before") {
```

Comment:
Hey, some time ago we agreed no to create icons this way. Icons should be moved to separate `<i>`

Reply:
fixed